BaseTools/Source/Python/Workspace/DscBuildData.py

```diff
@@ -1190,6 +1203,17 @@ def __ParsePcdFromCommandLine(self):
             for item in delete_assign:
                 GlobalData.BuildOptionPcd.remove(item)
 
+    def _ValidateLibraryClass(self, LibraryClass: str, LibraryInstance: PathClass) -> bool:
+        if LibraryClass =='NULL':

```

Comment:
nite: Space before `'NULL'`

```suggestion
        if LibraryClass == 'NULL':

```